sdk/src/main/scala/com/horizen/SidechainSettings.scala

```diff
@@ -39,7 +39,9 @@ case class WithdrawalEpochCertificateSettings(submitterIsEnabled: Boolean,
 
 case class ForgerSettings(automaticForging: Boolean = false,
                           restrictForgers: Boolean = false,
-                          allowedForgersList: Seq[ForgerKeysData] = Seq())
+                          allowedForgersList: Seq[ForgerKeysData] = Seq(),
+                          forgingFrequencyInSeconds: Int = 120
```

Comment:
fine for test. But not applicable for real environment, because must be set to some value for the whole network. If someone node will have a different value - will lead to divergence

Reply:
yes, this is something just for testing, the `tps-tests` branch will never be merged into dev